pages/landing/enterprise-eks/index.html

```diff
@@ -0,0 +1,191 @@
+          <li>Observability (Prometheus, Datadog, CloudWatch, ELK)</li>
+          <li>And more (cluster-autoscaler, sealed secrets, AWS Load Balancer Controller, etc)</li>
+      </ul>
+      On top of that, all our enterprise-grade EKS clusters are hardened according to the <a
+      href="https://aws.amazon.com/blogs/containers/introducing-cis-amazon-eks-benchmark/">CIS Amazon EKS
+      Benchmark</a>, regardless of the chosen technologies.
+    logos:
+      - title: CIS
+        src: /assets/img/logos-compliance/cis-benchmark.png
+      - title: EKS
+        src: /assets/img/logos-technologies/logo-eks.png
+
+  - heading: "Step 2: Deploy your EKS cluster"
+    tab_code_preview: site.data.enterprise-landing-page-tabs-code
+    image_side: left
+    body: |
+      Deploy your custom configured EKS cluster. Our friendly user interface can generate code for your existing AWS
+      accounts and VPCs, or you can dive right in to the code. Or, use <a href="https://gruntwork.io/landing-zone-for-aws/">Gruntwork Landing Zone</a> and the
+      <a href="https://gruntwork.io/reference-architecture">Gruntwork Reference Architecture</a> for a complete,
+      end-to-end stack, built with extensibility in mind. No matter what you choose, everything is managed as code with Terraform.
+    logos:
+      - title: Terraform
+        src: /assets/img/landing-heroku-page/terraform-icon.png
+      - title: Docker
+        src: /assets/img/landing-heroku-page/docker-icon.png
+      - title: Packer
+        src: /assets/img/landing-heroku-page/packer-icon.png
+
+  - heading: "Step 3: Set up your app"
+    image: /assets/img/landing-page-enterprise/sample-app-clip.gif
+    image_side: right
+    body: |
+      Use Gruntwork tools to optimize your enterprise apps for EKS. Update your app with cloud-native best practices for
+      packaging, monitoring, logging, service discovery, secrets management, schema migrations, and more. Or customize
+      the optimizations yourself using Gruntwork sample apps as a template. We include sample apps common enterprise languages and
+      frameworks:
+      <ul>
+          <li>Spring boot</li>
+          <li>Standard Java apps (e.g., Tomcat or similar)</li>
+          <li>Ruby on Rails</li>
+          <li>Python Django</li>
+          <li>Or suggest something!</li>
+      </ul>
+    logos:
+      - title: Spring Boot
+        src: /assets/img/logos-technologies/logo-spring-boot.png
+      - title: Rails
+        src: /assets/img/logos-technologies/logo-rails.png
+      - title: Django
+        src: /assets/img/logos-technologies/logo-django.png
+
+  - heading: "Step 4: Run your app"
+    tab_block: site.data.enterprise-landing-page-tabs-images
+    image_side: left
+    body: |
+      Gruntwork Enterprise EKS supports various deployment strategies out of the box, including canary, blue/green
+      deployments, automatic rollbacks, security scans, CI/CD, GitOps workflows, etc. Post deployment, use powerful
```

Comment:
The "post deployment" thing could be a separate step... with a focus on how easy the "day 2" operations are for devs and ops with the nice UI and built-in metrics, logs, etc.

Reply:
BTW, love the `.gif` for metrics. However, the one for logs is not as clear. It's a little too blurry to tell what's happening.

Also, sanity check: any image file must be less than 10MB, or the API Gateway we use in prod won't serve it. Check your `.gifs` and if they are too large, make them smaller (in terms of dimensions).

Reply:
> The "post deployment" thing could be a separate step... with a focus on how easy the "day 2" operations are for devs and ops with the nice UI and built-in metrics, logs, etc.

Separated it out to a separate step in the latest iteration.

> BTW, love the .gif for metrics. However, the one for logs is not as clear. It's a little too blurry to tell what's happening.

NOTE: these are placeholders based on some random youtube videos to give an idea of what we should place in these sections. We will want to think through what the actual thing to show here should be.

Reply:
I updated the gif here to replace the logs one with one showcasing argocd.